PODMAN.md

```diff
@@ -13,5 +13,11 @@ This will use `fuse-overlayfs` as storage layer. If the performance is not
 satisfactory, consider running podman as root to use plain `overlayfs2`:
 
 ```
-sudo podman system service -t 0 unix:///${HOME}/podman.sock
+mkdir -p $XDG_RUNTIME_DIR/podman
```

Comment:
This shouldn't be required AFAICT with rootless podman.

Reply:
It was required for me when running rootful podman manually.